src/extractor-lib/edfi_lms_extractor_lib/helpers/decorators.py

```diff
@@ -15,7 +15,7 @@ def callable_function(*args, **kwargs) -> bool:
             func(*args, **kwargs)
             return True
         except BaseException as e:
-            logger.exception("An exception occurred", e)
+            logger.exception("An exception occurred: %s", e)
```

Comment:
I'm not sure how we never noticed this before. The old line 18 actually throws an exception!